de.metas.business/src/main/java/org/adempiere/mm/attributes/api/impl/AttributeDAO.java

```diff
@@ -126,6 +125,19 @@ public I_M_AttributeValue retrieveAttributeValueOrNull(final I_M_Attribute attri
 	}
 
 	@Override
+	public I_M_AttributeValue retrieveAttributeValueOrNull_ForName(final I_M_Attribute attribute, final String name)
+	{
+
+		return Services.get(IQueryBL.class)
+				.createQueryBuilder(I_M_AttributeValue.class, attribute)
+				.addEqualsFilter(I_M_AttributeValue.COLUMN_M_Attribute_ID, attribute.getM_Attribute_ID())
```

Comment:
please *always* filter for active records, unless you have an explicit reason not to